cypress/integration/common/index.js

```diff
@@ -6,6 +6,11 @@ import { i18n } from '../../../src/utils';
 const baseURL = Cypress.config('baseUrl');
 
 Then('page title is {string}', (title) => {
+	Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
Why is the uncaught exception added here? If you need it for testing specific use case (e.g. for the API failure ), this step should be in the specific folder - otherwise, since the change is made in common folder, any tests that use this step will not fail, even though they should fail when there is no expected title

Reply:
I was trying to avoid creating another one specific error tests, and since the error would have been thrown before checking for the expected title. Agreed that should the step preceding this fail, then it wouldn't fail for non error specific tests. I'll add one for error pages.

Reply:
Updated!